src/compile.c

```diff
@@ -570,27 +586,35 @@ block gen_const_object(block expr) {
   jv k = jv_null();
   jv v = jv_null();
   for (inst *i = expr.first; i; i = i->next) {
-    if (i->op != SUBEXP_BEGIN ||
+    if (i->op == PUSH_UNDER) {
+      k = jv_copy(i->imm.constant);
+      i = i->next;
+    } else if (i->op != SUBEXP_BEGIN ||
         i->next == NULL ||
```

Comment:
I don't _think_ so, but I didn't want to rule out anything that was hand-bytecoded or that I'd otherwise missed.